packages/babel-plugin-import-jsx-pragma/src/index.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Default options for the plugin.
+ *
+ * @property {string}  scopeVariable Name of variable required to be in scope
+ *                                   for use by the JSX pragma. For the default
+ *                                   pragma of React.createElement, the React
+ *                                   variable must be within scope.
+ * @property {string}  source        The module from which the scope variable
+ *                                   is to be imported when missing.
+ * @property {boolean} isDefault     Whether the scopeVariable is the default
+ *                                   import of the source module.
+ */
+const DEFAULT_OPTIONS = {
+	scopeVariable: 'React',
+	source: 'react',
+	isDefault: true,
+};
+
+/**
+ * Babel transform plugin for automatically injecting an import to be used as
+ * the pragma for the React JSX Transform plugin.
+ *
+ * @see http://babeljs.io/docs/en/babel-plugin-transform-react-jsx
+ *
+ * @param {Object} babel Babel instance.
+ *
+ * @return {Object} Babel transform plugin.
+ */
+export default function( babel ) {
+	const { types: t } = babel;
+
+	let _options;
+
+	let hasJSX, hasImportedScopeVariable;
+
+	function getOptions( options ) {
+		if ( ! _options ) {
+			_options = {
+				...DEFAULT_OPTIONS,
+				...options,
+			};
+		}
+
+		return _options;
+	}
+
+	return {
+		visitor: {
+			JSXElement() {
+				hasJSX = true;
+			},
+			ImportDeclaration( path, state ) {
+				if ( hasImportedScopeVariable ) {
+					return;
+				}
+
+				const { scopeVariable, isDefault } = getOptions( state.opts );
+
+				// The module source isn't verified, since if at least the
```

Comment:
I think I get this comment but it's a bit hard to parse, especially because "since" could be "because" to make it read more logically.

I don't have ideas for copyediting right now but could this be explained another way? Or maybe broken up into a few sentences so it's easier to parse?